pkg/deploy-manager/config.go

```diff
@@ -100,6 +107,23 @@ func NewDeployManager() (*DeployManager, error) {
 		return nil, err
 	}
 
+	// rook ceph rest client
+	rookConfig, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	rookConfig.GroupVersion = &ocsv1.SchemeGroupVersion
+	rookConfig.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: codecs}
+	rookConfig.APIPath = "/apis"
+	rookConfig.ContentType = runtime.ContentTypeJSON
+	if rookConfig.UserAgent == "" {
+		rookConfig.UserAgent = rest.DefaultKubernetesUserAgent()
+	}
```

Comment:
I don't think this will be necessary as rook/ceph client pkg takes care of this.

Reply:
yeah makes sense. Removed this part.